unittest/app/app.js

```diff
@@ -14,20 +14,24 @@
 
 'use strict';
 
-const chai = require('chai');
-const chaiHttp = require('chai-http');
+import chai, {assert, expect} from "chai";
+import chaiHttp from "chai-http";
 chai.use(chaiHttp);
-const assert = chai.assert;
-const expect = chai.expect;
 
-const {app, version} = require('../../app');
+import fs from "fs-extra";
+import {fileURLToPath} from "url";
+
+import {app, version} from "../../app.js";
 const agent = chai.request.agent(app);
 
-const tests = Object.entries(require('../../tests.json'));
+const tests = Object.entries(JSON.parse(await fs.readFile(
```

Comment:
Can you extract out a `dirname` variable here and also use `fs.readJson()`?

Reply:
It looks like `fs-extra`, like NodeJS' built-in `fs`, accepts URL objects, so we can simplify this pretty easily!